shepherd-model/src/main/java/com/devsda/platform/shepherd/rabbitmq/RabbitMQOperations.java

```diff
@@ -0,0 +1,102 @@
+package com.devsda.platform.shepherd.rabbitmq;
+
+import com.rabbitmq.client.*;
+
+import javax.print.URIException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.util.concurrent.TimeoutException;
+
+public class RabbitMQOperations {
```

Comment:
Why two files of same code ?

Reply:
On Models I was testing it , Will remove this.